docs/how-to-guides/otel-collector.md

```diff
@@ -0,0 +1,283 @@
+    matchLabels:
+      app: json-app
+  template:
+    metadata:
+      labels:
+        app: json-app
+    spec:
+      terminationGracePeriodSeconds: 1
+      containers:
+        - name: json-app
+          image: busybox
+          command:
+            - "sh"
+            - "-c"
+            - |
+              while true; do
+                now=$(date -u '+%Y-%m-%dT%H:%M:%SZ')
+                echo "{\"message\":\"Hello world!\",\"level\":\"warn\",\"timestamp\":\"$now\"}"
+                sleep 1
+              done
+          resources:
+            limits:
+              memory: "64Mi"
+              cpu: "500m"
+            requests:
+              memory: "64Mi"
+              cpu: "500m"
+```
+
+Deploy this application via `kubectl apply -f apps.yaml`.
+
+Now we will set up a collector that can scrape logs from these apps, process them and send them to logfire.
+
+We'll need to store Logfire credentials somewhere, a Kubernetes Secret is a reasonable choice, a better choice for a production environment would be to use [External Secrets Operator](https://external-secrets.io/latest/).
+
+First create a Logfire write token, see [Create Write Tokens](./create-write-tokens.md).
+
+Now to save it as a secret in Kubernetes run the following command, replacing `your-write-token` with the value of the write token you just created:
+
+```shell
+kubectl create secret generic logfire-token --from-literal=logfire-token=your-write-token
+```
+
+Note that this is equivalent to the following `secrets.yaml` file, but using `kubectl` is easier because it will base64 encode the secret for you.
+
+```yaml title="secrets.yaml"
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logfire-token
+type: Opaque
+data:
+  logfire-token: base64-encoded-logfire-token
+```
+
+For the OTel Collector to scrape logs it will need permissions into the Kubernetes API which Kubernetes does not give out by default (you wouldn't want random pods being able to see logs from other pods by default!).
+
+To do this we'll create an `rbac.yaml` file with the following content:
+
+```yaml title="rbac.yaml"
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector-role
+rules:
+- apiGroups: [""]
+  resources: ["pods", "namespaces"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["apps"]
+  resources: ["replicasets"]
+  verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: otel-collector-role
+subjects:
+- kind: ServiceAccount
+  name: otel-collector
+  namespace: default
+```
+
+Apply this configuration via `kubectl apply -f rbac.yaml`.
+
+Now we can create the deployment for the collector itself.
+There are several options for deploying the OTel collector, including:
```

Comment:
perhaps worth mentioning [otel doc](https://opentelemetry.io/docs/platforms/kubernetes/collector/components)  as a reference to the deployments options